pkgs/top-level/all-packages.nix

```diff
@@ -40452,7 +40452,7 @@ with pkgs;
     jre = openjdk19; # TODO: remove override https://github.com/NixOS/nixpkgs/pull/89731
   });
 
-  owmods-cli = callPackage ../applications/misc/owmods-cli {
+  owmods-cli = callPackage ../games/owmods-cli {
```

Comment:
Please move the package to `pkgs/by-name/ow/owmods-cli/package.nix` and remove this line, thanks.